templates/customer/my-account.tpl

```diff
@@ -26,93 +26,80 @@
 
 {$componentName = 'customer-link'}
 
-{block name='page_content'}
-  <div class="row">
-    <div class="col-md-3 hidden-on-mobile">
-      {include file='components/account-menu.tpl'}
-    </div>
-
-    <div class="col-md-9">
-      <h1 class="h2">
-        {l s='Your personal information' d='Shop.Theme.Customeraccount'}
-      </h1>
-
-      <div class="{$componentName} row g-3">
-        <a class="{$componentName}__link col-12 col-md-4" id="identity-link" href="{$urls.pages.identity}">
-          <span class="link-item">
-            <i class="material-icons">&#xE853;</i>
-            {l s='Information' d='Shop.Theme.Customeraccount'}
-          </span>
-        </a>
-
-        {if $customer.addresses|count}
-          <a class="{$componentName}__link col-12 col-md-4" id="addresses-link" href="{$urls.pages.addresses}">
-            <span class="link-item">
-              <i class="material-icons">&#xE56A;</i>
-              {l s='Addresses' d='Shop.Theme.Customeraccount'}
-            </span>
-          </a>
-        {else}
-          <a class="{$componentName}__link col-12 col-md-4" id="address-link" href="{$urls.pages.address}">
-            <span class="link-item">
-              <i class="material-icons">&#xE567;</i>
-              {l s='Add first address' d='Shop.Theme.Customeraccount'}
-            </span>
-          </a>
-        {/if}
+{block name='page_title'}
+  {l s='Welcome' d='Shop.Theme.Customeraccount'} {$customer.firstname} {$customer.lastname}
+{/block}
 
-        {if !$configuration.is_catalog}
-          <a class="{$componentName}__link col-12 col-md-4" id="history-link" href="{$urls.pages.history}">
-            <span class="link-item">
-              <i class="material-icons">&#xE916;</i>
-              {l s='Order history and details' d='Shop.Theme.Customeraccount'}
-            </span>
-          </a>
-        {/if}
+{block name='page_content'}
+  <div class="{$componentName} row g-3">
+    <a class="{$componentName}__link col-md-6 col-lg-4" id="identity-link" href="{$urls.pages.identity}">
```

Comment:
Let's rethink this naming to be short and clean.
Also, I think we can move `<span class="link-item">` and wrap only the text with it. I don't like mixing text with other elements, wdyt?